lib/modules/internmodule/internmodule.py

```diff
@@ -37,10 +37,9 @@ def interpret_message(self, msg, email, db, admin) -> Dict:
                 else:
                     return catch_incorrect_arguments(Commands.INTERN_REMOVE)
             if commands[0] == 'list':
-                if len(commands) <= 2:
+                if len(commands) == 1:
```

Comment:
this code is confusing, is there a way you could easily retructure it to be simpler?